README.md

```diff
@@ -36,9 +36,9 @@ Terraform configuration.
 
 #### Block
 
-If your configuration is using a nested block to define timeouts, such as the following:
+The following illustrates nested block syntax for defining timeouts on a resource and a data source.
 
-```terraform
+```hcl
```

Comment:
Using `terraform` for code blocks should be valid in GitHub: https://github.com/github/linguist/blob/c1c34e5260797b4d598f5ec76f19723bfc5a1894/lib/linguist/languages.yml#L2529-L2544 (and especially helpful should the language ever be split off from generic hcl formatting)